Src/BootCamp.Chapter/BMISLeslie

```diff
@@ -0,0 +1,38 @@
+using System;
+
+namespace BMISLeslie
+{
+    class Program
+    {
+        private static void Main(string[] args)
+        {
+            //Declare variables
+            string name;
+            string surname;
+            int age; 
+            decimal height, weight;
+            decimal bmi;
+            string printOut;
+
+            //Console Entries and add value to variables
+            Console.WriteLine("Please enter your first name:");
+            name = (Console.ReadLine());
```

Comment:
You can declare and assign a variable at once. Like the following: 
`string name = Console.ReadLine()`.

Btw, you don't need to wrap `Console.ReadLine()` into the brackets.